src/NHSD.BuyingCatalogue.Smtp/stunnel.conf

```diff
@@ -0,0 +1,6 @@
+[smtp]

```

Comment:
Same as above, we could get rid of the whole .Smtp directory

Reply:
> Good work, few things to address.
> 
> Also - I imagine we'd need to amend our chart to accommodate for bringing the smtp server in, can we please do it as part of this PR?

I've created a separate task for this [9450](https://buyingcatalog.visualstudio.com/Buying%20Catalogue/_workitems/edit/9450). Put a message in the platform channel. Would be good to work out whether or not some of the settings can be reused.

Reply:
Ah, sorry, didn't see your broke this off into its own task, my bad. As far as I know most, if not all, of the smtp settings will be reusable.